src/components/structures/RightPanel.tsx

```diff
@@ -199,7 +199,9 @@ export default class RightPanel extends React.Component<IProps, IState> {
             }
             case RightPanelPhases.Room3pidMemberInfo:
             case RightPanelPhases.Space3pidMemberInfo:
-                card = <ThirdPartyMemberInfo event={cardState?.memberInfoEvent} key={roomId} />;
+                if (cardState?.memberInfoEvent) {
```

Comment:
I don't think this is allowed by our coding style, which forbids "relying on implicit truthiness of non-boolean values".

(I don't make the rules...)

Reply:
likewise lots of places throughout this PR.